docs/interpreter/livy.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: "Livy Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+## Livy Interpreter for Apache Zeppelin
+Livy is an open source REST interface for interacting with Spark from anywhere. It supports executing snippets of code or programs in a Spark context that runs locally or in YARN.
+
+* Interactive Scala, Python and R shells
+* Batch submissions in Scala, Java, Python
+* Multi users can share the same server (impersonation support)
+* Can be used for submitting jobs from anywhere with REST
+* Does not require any code change to your programs
+
+### Requirements
+
+Additional requirements for the Livy interpreter are:
+
+ * Spark 1.3 or above.
+ * Livy server.
+
+### Configuration
+<table class="table-configuration">
+  <tr>
+    <th>Property</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+      <td>zeppelin.livy.master</td>
+      <td>local[*]</td>
+      <td>Spark master uri. ex) spark://masterhost:7077</td>
+    </tr>
+  <tr>
+    <td>zeppelin.livy.url</td>
+    <td>http://localhost:8998</td>
+    <td>URL where livy server is running</td>
+  </tr>
+  <tr>
+    <td>zeppelin.livy.spark.maxResult</td>
+    <td>1000</td>
+    <td>Max number of SparkSQL result to display.</td>
+  </tr>
+</table>
+
+
+
+## How to use
+Basically, you can use
+
+**spark**
```

Comment:
Currently this part is shown like below.
![screen shot 2016-05-19 at 3 18 17 pm](https://cloud.githubusercontent.com/assets/10060731/15384381/3d9acf64-1dd5-11e6-96bd-b47615e582d1.png)

If you intended this :
![screen shot 2016-05-19 at 3 18 32 pm](https://cloud.githubusercontent.com/assets/10060731/15384395/5b07f928-1dd5-11e6-8888-23f842d1a5d8.png)

You need to add a newline in here :)